pkg/config/config.go

```diff
@@ -349,6 +353,39 @@ func (dc *DeploymentConfig) listUnusedModules() map[string][]string {
 	return unusedModules
 }
 
+func (dc *DeploymentConfig) listUnusedDeploymentVariables() []string {
+	var usedVars = []string{}
```

Comment:
Why not `map[string]bool` ? No need to use `slices.Contains` then.

Reply:
Addressed.